chpl/chpl-service/src/main/java/gov/healthit/chpl/svap/domain/CertificationResultSvap.java

```diff
@@ -22,6 +24,7 @@
  *
  */
 @XmlType(namespace = "http://chpl.healthit.gov/listings")
+@XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
```

Comment:
What is this for? Should this be standard?

Reply:
This is often put in package-info.java in our code, but this particular package does not package-info.java.  Often, package-info.java is used for JavaDoc documentation about a package.  You can also specify annotations that apply to all classes in the package.  I'm not sure if we really have a standard for this.

Reply:
I guess I'm asking why we need it now. We've not used it in any of our other XML domain definition documents, so what's the purpose here? If this is a good thing to add, then should we add it to all of our things? If it's superfluous or otherwise "extra", then why do we have it?

Reply:
And if it is usually done in the package-info file, and is needed, then why isn't it in that file here?

Reply:
If we have a standard for how we get these things to be alphabetized, and that standard is the package-info.java file, then we should stick to it.

Reply:
Updated...  In the future, do we want to move these to `package-info`? I think it is a very similar situation. 
![image](https://user-images.githubusercontent.com/24817202/104483617-51c12580-5596-11eb-966d-058204527651.png)